pkg/config/constants/constants.go

```diff
@@ -166,11 +166,13 @@ const (
 	WaypointServiceAccount = "istio.io/for-service-account"
 	WaypointForAddressType = "istio.io/waypoint-for"
 
-	ManagedGatewayLabel               = "gateway.istio.io/managed"
-	UnmanagedGatewayController        = "istio.io/unmanaged-gateway"
-	ManagedGatewayControllerLabel     = "istio.io-gateway-controller"
```

Comment:
We donot apply this label on kube-gateway,  not sure should we use it later? @howardjohn